.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -1,54 +1,55 @@
+title: 'Request creation of a new format specification'
 labels: spec-request
 ---
 <!--
-A request for a format specification consist of 2 YAML blocks followed by a free-form text description.
-Fill in all the fields possible. Delete the fields without data and all the comments from this template (including this one!).
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete fields you cannot populate and all the comments from this template (including this one!).
 The resulting YAML blocks should contain valid YAML.
+Mandatory fields are marked with the word MANDATORY for your convenience.
 -->
 ```yaml
-# This block is the boilerplate of a KSY spec file. Must be a valid KSY. Meant to be ready to be copied as it is.
-meta:
-  id: # insert an id here. KS currently has no namespaces, so the id MUST be unique within all the repo and this property should be futureproof.
-  title: # insert format human-readable name here
-  application: # Insert the application human-readable name here
+# This block is the boilerplate of a KSY spec file. It must be a valid KSY. It is meant to be ready to be copied as it is in order to make the task of the implementer easier.
+meta: # see https://doc.kaitai.io/ksy_reference.html#meta for more info
+  id: # insert an ID you feel right to assign to the format here. KS currently has no namespaces, so the id MUST be unique within all the repo and you should be sure there will be no collisions. If in doubt, add a prefix of an app name. MANDATORY.
+  title: # insert a human-readable name of the format here. MANDATORY.
+  application: # insert a human-readable name of the application for which this format is native here. If it is native for several applications, choose the ones you feel the most affiliated with the format. If the format is a widespread one, like PNG or WAV, just remove this field.
   file-extension:
-    # a YAML list of file extensions here
-  encoding: UTF-8
-  endian: le
-  xref: # check if the following sources contain info about the format you request
+    # insert here a YAML list of file extensions should be placed here.
+  xref: # check if the following sources contain info about the format you request. See https://doc.kaitai.io/user_guide.html#meta for more info
     mime: # MIME type
     rfc: # Link to RFC
+    din:
+      id: # DIN standard identifier
+    iso: # ISO standard identifier
     forensicswiki: # Page name on https://forensicswiki.org/
     justsolve: # ID on http://fileformats.archiveteam.org/wiki/
     pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
      - x-fmt/ 
     wikidata: Q # ID on WikiData
     fileformat: # Link to https://www.fileformat.info
-    loc: # ID in https://www.loc.gov/preservation/digital/formats/
-    din:
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/    din:
       id: # DIN standard identifier
```

Comment:
This line seems to be left over from moving the `din` block up.

Also the `xref` keys should be sorted alphabetically, as indicated in the style guide.

Reply:
Fixed.